bgzf.c

```diff
@@ -1123,3 +1319,27 @@ long bgzf_utell(BGZF *fp)
     return fp->uncompressed_address;    // currently maintained only when reading
 }
 
+static const char * bgzf_zerr(int errnum)
+{
+    static char buffer[32];
+    // gzerror OF((gzFile file, int *errnum)
+    switch (errnum) {
+    case Z_ERRNO:
+        return strerror(errno);
+    case Z_STREAM_ERROR:
```

Comment:
Probably covered under "inconsistent stream state"?  The text for this one is a bit of a mess as Z_STREAM_ERROR can have different meanings depending on which function returned it.